stdlib/public/core/VarArgs.swift

```diff
@@ -60,7 +60,7 @@ protocol _CVarArgAligned : CVarArg {
 
 #if arch(x86_64)
 @_versioned
-internal let _x86_64CountGPRegisters = 6
+internal let CountGPRegisters = 6
```

Comment:
Internal variables should be named with a leading underscore and using `camelCase`: this should be `countGPRegisters`.